experimental/gittuf/policy_test.go

```diff
@@ -114,3 +114,18 @@ func TestPullPolicy(t *testing.T) {
 		assert.ErrorIs(t, err, ErrPullingPolicy)
 	})
 }
+
+func TestApplyPolicy(t *testing.T) {
+	localRepo := createTestRepositoryWithPolicy(t, "")
+	err := localRepo.ApplyPolicy(testCtx, false)
```

Comment:
isn't the policy already applied by the helper?

Reply:
Thank you for your feedback. Yes, it seems like `ApplyPolicy` simply makes a call to `policy.Apply` which is already done inside `createTestRepositoryWithPolicy`. 
Do you think I could change the `createTestRepositoryWithPolicy` function in helpers_test.go to call the `ApplyPolicy` function instead, for coverage?

Reply:
I think it might be worth moving it out of this PR.